canvas_modules/common-canvas/src/common-properties/controls/radioset/radioset.scss

```diff
@@ -29,12 +29,31 @@
 		display: inline-flex;
 		flex-wrap: wrap;
 		.properties-radioset-panel {
+			display: flex;
+			height: auto;
+			align-items: center;
 			margin-bottom: 0;
 		}
 	}
 	.properties-radioset-panel {
 		margin-bottom: $spacing-02;
 		margin-right: $spacing-04;
+		display: grid;
+		grid-template-columns: 1fr 8fr;
+		grid-template-rows: auto;
+		grid-template-areas:
+			"radio tooltip"
+			"panel panel";
```

Comment:
why is panel here twice?

Reply:
i want `properties-control-nested-panel` which has panel text container stretch all the way and not overlap with others
<img width="636" alt="image" src="https://user-images.githubusercontent.com/89609176/177407818-c9904ba2-794a-4cb4-af5f-cb7706f992da.png"> this is when i left only panel grid area item 
<img width="351" alt="image" src="https://user-images.githubusercontent.com/89609176/177408260-bbb030f4-eaef-4a3f-b82c-cf79f9f9177b.png">
<img width="351" alt="image" src="https://user-images.githubusercontent.com/89609176/177408517-e18adb8a-bcb2-482e-90d0-43b7ba1da33d.png">

Reply:
I see.  It required 2 fields to match line 1 and having `panel panel` make it so it stretches over both grids.  This works.